lib/constants/markets.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: Review this number as max safe doesn't work
+export const MAX_MARKET_PRICE = 100000000;
```

Comment:
From Max in Telegram:

> i think the quickest way forward would be to send i64::MAX for quote lots, if you want to limit it properly, use: ceil(baseLots * priceLots * (1+max(maker_fee, taker_fee)*1e-6)) + 1

I know this is somewhere else, but since this is max...

Reply:
not sure if this is what he was talking about but I think you are right.